gogio/androidbuild.go

```diff
@@ -446,6 +448,7 @@ func exeAndroid(tmpDir string, tools *androidTools, bi *buildInfo, extraJars, pe
 		Features:    features,
 		IconSnip:    iconSnip,
 		AppName:     appName,
+		HasService:  stringsContains(permissions, foregroundPermission),
```

Comment:
I think it's more canonical to search for `"foreground"` in `perms`.

Reply:
changed.